_locales/en/messages.json

```diff
@@ -646,6 +646,12 @@
     "configurationBatteryScale": {
         "message": "Voltage Scale"
     },
+    "configurationVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "configurationVbatPidCompensationHelp": {
+        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
```

Comment:
I had a quick look in `src/main/sensors/battery.c`, and it looks like only `batteryConfig->vbatmaxcellvoltage` is currently used, and it's lower constrained by `maxCalculatedVoltage - batteryConfig->vbatmaxcellvoltage`. Is this temporary, or should I make the help text indicate that the max cell voltage is used?